src/Servers/Connections.Abstractions/ref/Microsoft.AspNetCore.Connections.Abstractions.netstandard2.1.cs

```diff
@@ -196,15 +235,14 @@ public partial interface IMemoryPoolFeature
     {
         System.Buffers.MemoryPool<byte> MemoryPool { get; }
     }
-    public partial interface IQuicStreamFeature
+    public partial interface IProtocolErrorCodeFeature
     {
-        bool CanRead { get; }
-        bool CanWrite { get; }
-        long StreamId { get; }
+        long Error { get; set; }
     }
-    public partial interface IQuicStreamListenerFeature
+    public partial interface IStreamDirectionFeature
     {
-        System.Threading.Tasks.ValueTask<Microsoft.AspNetCore.Connections.ConnectionContext> AcceptAsync();
+        bool CanRead { get; }
+        bool CanWrite { get; }
```

Comment:
Hey, for once something `Stream` already has :P

Reply:
You need to be able to pass this in when creating a Stream to indicate Read/Write. So Stream doesn't give you _everything_ 😄